Microsoft/action_change_user_password.json

```diff
@@ -0,0 +1,51 @@
+{
+  "arguments": {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "properties": {
+      "server": {
+        "description": "Remote server dns name or IP address",
+        "type": "string"
+      },
+      "username": {
+        "description": "Username to do authentication on remote server",
+        "type": "string"
+      },
+      "password": {
+        "description": "Password to do authentication on remote server",
+        "type": "string"
+      },
```

Comment:
I think that these parameters should be declared as module parameters in the manifest.json